new-lamassu-admin/src/pages/Maintenance/Wizard/Wizard.js

```diff
@@ -12,6 +12,8 @@ const MODAL_WIDTH = 554
 const MODAL_HEIGHT = 520
 const CASHBOX_DEFAULT_CAPACITY = 500
 
+const CASSETTE_FIELDS = ['cassette1', 'cassette2', 'cassette3', 'cassette4']
```

Comment:
Hmm, we should create the cassette fields based on the numberOfCassettes instead of hardcoding this, right?